ci/pipelines/Build.jenkinsfile

```diff
@@ -0,0 +1,97 @@
+def channel = "testing"
+def version = "0.0.0"
+def packageName = "DbSQLiteAdapter"
```

Comment:
I like the idea of declaring some vars to be used on the pipeline. This facilitates reusing the code from different repos, so I will start doing so too.